<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Microbiome@GitHub</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    
  </head>
  
  <body>
    <div class="wrapper">
      <header>
        <h1>miaverse</h1>
        <p>R/Bioconductor ecosystem for microbiome bioinformatics based on (Tree)SummarizedExperiment.</p>

	<p></p>

	<p>Key developers:</p>
	
	Felix G.M. Ernst<br/>
	Ruizhu Huang<br/>		
        Sudarshan Shetty<br/>
	Tuomas Borman<br/>	
	Domenick C. Braccia<br/>
        Hector Corrada Bravo<br/>
	<a href="http://www.iki.fi/Leo.Lahti">Leo Lahti</a>

	<p></p>

        <p>The full list of contributors varies over time, and is
        identified through the individual repositories. Check each
        individual package for acknowledgements, contact information
        and references. Don't hesitate to contact us for feedback,
        support, or feature requests.</p>

	<p></p>
	

	
      </header>

      <section>


<h2>miaverse: R/Bioconductor framework for microbiome data science</h2>

<p>We are envisioning to rethink microbiome data science in R/Bioconductor.</p>

<p>This is an open source project developing R/Bioc methods,
benchmarking data, and educational material for microbiome research
based on the SummarizedExperiment class and its derivatives. New
contributions are very welcome. The current project brings together
the following key elements:</p>

<h3>Data</h3>

<ul>

  <li><b>Data container</b> (Bioc): <a href="https://doi.org/doi:10.18129/B9.bioc.TreeSummarizedExperiment">TreeSummarizedExperiment</a> class</li>

  <li><b>Data resources</b> (Bioc): <a href="https://bioconductor.org/packages/devel/data/experiment/html/microbiomeDataSets.html">microbiomeDataSets</a>: Experiment Hub based benchmarking/demonstration data sets.</li>

</ul>


<h3>Methods (R packages)</h3>


    <ul>
      <li><a href="https://bioconductor.org/packages/devel/bioc/html/mia.html">mia</a> for microbiome data analysis and manipulation</li>
      <li><a href="https://bioconductor.org/packages/devel/bioc/html/miaViz.html">miaViz</a> for microbiome visualization</li>
      <li>External packages supporting the framework</li>
    </ul>


    
<h3>Documentation</h3>
    


<a href="https://microbiome.github.io/OMA/">Orchestrating Microbiome Analysis with R/Bioconductor</a> manual (beta version; under preparation)  

  
<h3>Community</h3>

Join us at the following channels

<ul>
<li><a href="https://bioc-community.herokuapp.com/">Bioconductor Slack</a> (#miaverse channel)</li>
<li><a href="https://gitter.im/microbiome/miaverse?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge">Gitter/Matrix</a> (#miaverse channel)</li>
</ul>


The project was introduced
in <a href="https://eurobioc2020.bioconductor.org/">European
Bioconductor Meeting (EuroBioC2020)</a>, and it is linked to the
European COST action network
on <a href="https://www.ml4microbiome.eu/">Statistical and Machine
Learning Techniques in Human Microbiome Studies</a>.


<p></p>

<h2>Dissemination and training material</h2>


<ul>
  <li><a href="https://f1000research.com/slides/10-748">Bioc2021 slides</a></li>
  <li><a href="https://f1000research.com/posters/9-1459">EuroBioc2020 poster</a></li>
  <li><a href="https://f1000research.com/slides/9-1464">EuroBioc2020 slides</a></li>    
  <li><a href="https://microbiome.github.io/course_2021_radboud">Radboud summer course 2021</a></li>
  <li>Huang R, Soneson C, Ernst FGM et al. TreeSummarizedExperiment: a S4 class for data with hierarchical structure. F1000Research 2021, 9:1246 <a href="https://doi.org/10.12688/f1000research.26669.2">doi:10.12688/f1000research.26669.2</a></li>
  
</ul>

<p></p>


	<center>
	<img src="Rlogo.png", width="15%"></img>
	<img src="logo_bioconductor.gif", width="25%"></img>		  
	<img src="eurobioc2020.png", width="10%"></img>
	<img src="ml4microbiome_logo.png", width="30%"></img>
	</center>


<!--

<h3>Related resources</h3>

<p>The <a href="https://github.com/microbiome">Github site</a> lists
also other tools for microbiome analysis:</p>

<ul>
  
  <li>The <a href="https://microbiome.github.io/microbiome">microbiome R package</a> extends the <a href="https://joey711.github.io/phyloseq/">phyloseq data format</a> and includes a comprehensive <a href="http://microbiome.github.io/microbiome/">tutorial</a> for large-scale microbiome analysis and visualization</li>
  
    <li>The <a href="https://github.com/microbiome/earlywarningtoolbox">early warnings toolkit</a> to analyze shifts between alternative stable states in complex (eco)systems</li>
  
  <li>The <a href="https://github.com/hallucigenia-sparsa/seqtime">seqtime</a> for time series analysis
    
  <li>The <a href="https://github.com/microsud/microbiomeutilities">microbiomeutilities</a> for microbiome data wrangling

  <li><a href="https://github.com/microbiome/NMGS/">NMGS</a> neutral models for microbial metagenomics</li>

  <li><a href="https://github.com/microbiome/RPA/">RPA</a> Robust Probabilistic Averaging</li>
  
  <li><a href="https://github.com/BinPro/CONCOCT">CONCOCT</a> Binning metagenomic contigs by coverage and composition 
  

</ul>

-->

      </section>

    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>

