<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Microbiome@GitHub</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>Microbiome</h1>
        <p>Open source for microbiome bioinformatics</p>

        <!--<p class="view"><a href="https://github.com/microbiome/">View the GitHub Projects <small>microbiome/</small></a></p>-->

        <p class="view"><p>Contact: <a href="http://www.iki.fi/Leo.Lahti">Leo Lahti</a><br/><!--<small>Hosted on GitHub Pages</small></a></p>-->
        
        <p>Key
        developers/contributors include Felix G.M. Ernst, Domenick
        C. Braccia, Ruizhu Huang, Tuomas Borman, Hector Corrada Bravo,
        Vasilis Dakos, Christopher Quince, Karoline Faust, Sudarshan
        Shetty, and others.</p>

        <p>The full list of contributors varies over time, and is
        identified through the individual repositories. Check each
        individual package for acknowledgements, contact information
        and references. Don't hesitate to contact us for feedback,
        support, or feature requests.</p>


	<img src="eurobioc2020.png"></img>
      </header>

      <section>



<h3>MicrobiomeExperiment for R/Bioc</h3>

The MicrobiomeExperiment is an emerging ecosystem of R classes,
methods, and educational material for microbiome research, formally
introduced
in <a href="https://eurobioc2020.bioconductor.org/">European
Bioconductor Meeting (EuroBioC2020)</a>. This work is now under active
development as a joint work by multiple teams. New contributors and
early adopters are very welcome. A greater project landing page will
follow later :-) </p>

<p>We are envisioning to rethink the microbiome research in
R/Bioc. The new ecosystem contains the following key elements:</p>


<ul>

  <li><b>Class</b> (Bioc): <a href="https://github.com/HCBravoLab/MicrobiomeExperiment">MicrobiomeExperiment</a></li>
  
  <li><b>Methods</b> (R packages): <a href="https://github.com/FelixErnst/mia">mia</a> for analysis, <a href="https://github.com/microbiome/miaViz">miaViz</a> for visualization, other tools are on the way</li>
  
  <li><b>Documentation:</b> <a href="https://github.com/microbiome/MiaBook">MiaBook</a> manual</li>

  <li><b>Community:</b> join us at MicrobiomeExperiment channel in <a href="https://bioc-community.herokuapp.com/">Bioconductor Slack</a></li>  
  
</ul>


For further background, see

<ul>

  <li><a href="https://f1000research.com/posters/9-1459">EuroBioc2020 poster</a></li>

  <li><a href="https://f1000research.com/slides/9-1464">EuroBioc2020 slides</a></li>  

</ul>

<p></p>

<h3>Related resources</h3>

<p>The <a href="https://github.com/microbiome">Github site</a> lists
several tools for microbiome analysis that are on the way to being
adapted to the new class structure, including:</p>

<ul>
  
  <li>The <a href="https://microbiome.github.io/microbiome">microbiome R package</a> extends the <a href="https://joey711.github.io/phyloseq/">phyloseq data format</a> and includes a comprehensive <a href="http://microbiome.github.io/microbiome/">tutorial</a> for large-scale microbiome analysis and visualization</li>
  
    <li>The <a href="https://github.com/microbiome/earlywarningtoolbox">early warnings toolkit</a> to analyze shifts between alternative stable states in complex (eco)systems</li>
  
  <li>The <a href="https://github.com/hallucigenia-sparsa/seqtime">seqtime</a> for time series analysis
    
  <li>The <a href="https://github.com/microsud/microbiomeutilities">microbiomeutilities</a> for microbiome data wrangling

  <li><a href="https://github.com/microbiome/NMGS/">NMGS</a> neutral models for microbial metagenomics</li>

  <li><a href="https://github.com/microbiome/RPA/">RPA</a> Robust Probabilistic Averaging</li>
  
  <li><a href="https://github.com/BinPro/CONCOCT">CONCOCT</a> Binning metagenomic contigs by coverage and composition 
  

</ul>


<!--

<h3>Data sets</h3>

<ul>

  <li><a href="http://datadryad.org/resource/doi:10.5061/dryad.pk75d">HITChip Atlas</a> with 1006 normal western adults; mostly cross-sectional, including some time sereies and comprehensive metadata</li>

  <li><a href="http://datadryad.org/resource/doi:10.5061/dryad.1mn1n">Diet swap study</a> between African Americans and Rural Africans</li>

  <li><a href="https://peerj.com/articles/32/">Gut microbiota and blood serum lipidomics</a></li> 

  <li>Example data sets in the <a href="https://github.com/microbiome/microbiome/blob/master/vignettes/Data.md">microbiome R package</a> 
  
</ul>


-->

<!--

<h3>Presentation slides and publications</h3>

<ul>

  <li><a href="20160901-Lahti.pdf">MicrobesRUs seminar, Turku, Finland Sep 1, 2016</a> 
  
  <li><a href="http://antagomir.github.io/work/pubs/">Publications</a> 
  
</ul>


-->

      </section>
<!--
      <footer>
        <p>Maintained by <a href="http://www.iki.fi/Leo.Lahti">Leo Lahti</a>; 
        <small>Hosted on GitHub Pages</small></p>
      </footer>
-->
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>

